Formula/dmd.rb

```diff
@@ -98,6 +87,9 @@ def install
     cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
     lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
 
+    system "which", "dmd"
```

Comment:
that was in the commits marked `DO NOT MERGE`, I just added them to get some debug output from the auto-tester